src/back-end/lib/db/proposal/sprint-with-us.ts

```diff
@@ -1060,7 +1053,6 @@ export const awardSWUProposal = tryDb<[Id, string, AuthenticatedSession], SWUPro
             status: SWUProposalStatus.NotAwarded,
             note: ''
           });
-      }
```

Comment:
@BCerki on 1029 the proposal record of the winning proposal gets its `updatedAt` field updated with the `now` variable created on 1014. Can we also update all of the proposal records in `otherProposalIds` `updatedAt` and `updatedBy` fields? should be bulkable.

Reply:
I checked in the db that the the `updatedAt` day is today and the `updatedBy` is the id of the test admin user from the fixture:
```
digitalmarketplace=# select * from "swuProposals";
-[ RECORD 1 ]----------+-------------------------------------
id                     | 7a6dce2b-8344-41aa-a429-43e80758aef2
createdAt              | 2022-02-18 18:59:35.676+00
createdBy              | 06172f56-f08c-4380-86db-603bec98a8c7
updatedAt              | 2022-02-25 22:12:08.666+00
updatedBy              | 39b86e99-9a39-49ae-8962-c7d815e56126
challengeScore         | 61
scenarioScore          | 62
priceScore             | 100
opportunity            | 593f191a-8c7d-4a3b-92e2-4703a80f4f05
organization           | 0e593592-4df5-412c-bd68-cc7718e3bc75
anonymousProponentName | Proponent 2
-[ RECORD 2 ]----------+-------------------------------------
id                     | 7a6dce2b-8344-41aa-a429-43e80758aef1
createdAt              | 2022-02-18 18:59:35.676+00
createdBy              | 06172f56-f08c-4380-86db-603bec98a8c4
updatedAt              | 2022-02-25 22:12:08.666+00
updatedBy              | 39b86e99-9a39-49ae-8962-c7d815e56126
challengeScore         | 31
scenarioScore          | 
priceScore             | 
opportunity            | 593f191a-8c7d-4a3b-92e2-4703a80f4f05
organization           | 0e593592-4df5-412c-bd68-cc7718e3bc74
anonymousProponentName | Proponent 1
```